mhs/README.md

```diff
@@ -50,7 +50,17 @@ MHS takes a number of environment variables when it is run. These are:
 * `MHS_SYNC_ASYNC_STORE_RETRY_DELAY` (inbound only) The delay in milliseconds between retrying placing a message on the sysnc-async store. Defaults to 100ms
 * `MHS_RESYNC_RETRIES` (outbound only) The total number of attempts made to the sync-async store during resynchronisation, defaults to 20
 * `MHS_RESYNC_INTERVAL` The time in between polls of the sync-async store, the interval is in seconds and defaults to 1
-
+* `MHS_SPINE_ROUTE_LOOKUP_URL` (outbound only) The URL of the Spine route lookup service. E.g `https://example.com`. This URL should not contain path or query parameter parts.
+* `MHS_SPINE_ORG_CODE` (outbound only) The organisation code for the Spine instance that your MHS is communicating with. E.g `YES`
+* `MHS_SDS_URL` (Spine Route Lookup service only) The URL to communicate with SDS on. e.g. `ldaps://example.com`
+* `MHS_DISABLE_SDS_TLS` (Spine Route Lookup service only) A flag that can be set to disable TLS for SDS connections.
+*Must* be set to exactly `True` for TLS to be disabled.
+* `MHS_SDS_CACHE_IMPLEMENTATION` (Spine Route Lookup service only) Identifies the caching implementation to use when
+caching values retrieved from SDS. This value is optional. Possible values are:
+    * `DICTIONARY_CACHE` - An in-memory cache using a Python dictionary. Not suitable for production use. If no
```

Comment:
The idea was that each implementation of the SDS cache interface will register itself in a dictionary, and then the desired implementation can be loaded from there based on a config key. It's a pattern I'd discussed with Dave previously, which I thought made sense to use here since we plan to implement a second implementation of the SDS cache interface imminently

See https://github.com/nhsconnect/integration-adaptors/pull/68/commits/76825f249aa37cba6694653ced12235647cef94c for the implementation.

Reply:
Sorry I just noticed that `DICTIONARY_CACHE` is indented, and thus being one of the expected values of `MHS_SDS_CACHE_IMPLEMENTATION`, that makes perfect sense now 👍